app/models/candidate.rb

```diff
@@ -15,7 +15,9 @@ class Candidate < ApplicationRecord
   def current_application
     choice = ApplicationChoice.first_or_create do |ac|
       provider = Provider.find_or_create_by(code: 'ABC') { |p| p.name = 'Example provider' }
-      ac.course = Course.find_or_create_by(name: 'English Primary', code: '123', provider: provider)
+      course = Course.find_or_create_by(name: 'English Primary', code: '123', provider: provider, level: 'primary')
+      site = Site.find_or_create_by(code: 'A', name: 'Example School', provider: provider)
+      ac.course_option = CourseOption.find_or_create_by(course: course, site: site, vacancy_status: 'B')
```

Comment:
I know this was partially there, should we avoid to have all these magic strings and numbers coded in the class? (I did not get what is that current application_choice that it creates, to be honest)

Reply:
@alexdesi this stuff is there to allow end-to-end testing. This will be removed once the course choice feature is implemented.

Reply:
@tijmenb great, maybe we should add a comment for it, thanks!